.vscode/launch.json

```
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    // use gdb in linux and lldb in mac
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(lldb) Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/src/bin/lfortran",
+            "args": ["a.f90"],
```

Maybe use some fail that exists?
```suggestion
            "args": ["examples/expr2..f90"],
```